### ISSUE TITLE
Run 2.0i preflight checklist

### ISSUE THREAD

**Issue by cwwalter** [LSSTDESC/imSim#116]
There are a few items we either need or should square away before the 2.0 run.  These are not yet needed for 1.2i so that test can proceed with out these if necessary:

- [x] When the static lens sources are ready (which are presented as postage stamps), we need to add the image input type and call the interpolated image class in GalSim.  (#83).
- [x] We should make sure we get at least a ball park estimate from the project for the cosmic rate.  We have images from a few places we are using for studies to really estimate this.  But just being close is probably OK now.  The SN group wants to study false positive rates so we actually care about the value.  This is issue #87. The rate now (set in the config files) is 8 CRs/sec/sensor.  CWW will ask if there is project guidance.
- [x] Merge DCR GalSim surface operation branch (#65)
- [x] Merge Centroid branch (#32)
- [x] Fix bleed trail wrap around (#153)
- [x] Remove dependence on obs_lsstSims (#121)
- [x] Validate the RandomWalk galaxies (#131)
- [x] Do GalSim 2.0 consistency check (#154).
- [x] Update sensor properties, putting them in the lsstCam YAML files (#130).
- [x] ~~Run sample images with all configs turned on for validation team~~(Ran out of time).

These issues are related to workflows and while not strictly necessary they are targeted to be done before we start 2.0i.

- [x] Write centroid files in .gz format (#149)
- [x] Estimate expected run times based on instance catalogs (#148)
- [x] Add an option to skip e-image production and only produce amp files (#146)
- [x] Make checkpoint summary information more easily available  to the workflow system (#147)

**Comment by cwwalter** [comment]
Add merging DCR GalSim surface operation and centroid writing to the 2.0i preflight checklist.  Both of these PRs should be ready to be merged.

**Comment by cwwalter** [comment]
Add removing obs_lsstSims dependence (#121) and adding I&T measured sensor parameters to obs_lsstCam policy files (#130) to the 2.0i preflight checklist.

**Comment by cwwalter** [comment]
Add validation of the randomWalk complex morphology galaxies to the 2.0i preflight checklist.

**Comment by cwwalter** [comment]
Add back-porting of diffusion coefficient corrections described in #141 to the 2.0i preflight checklist.

**Comment by katrinheitmann** [comment]
Thanks Chris! It would be great if we have a very clear plan for this 
last test early on (meaning with respect what visits to redo etc) so 
that we basically only will have to hit a run button. Hopefully the 
results we have from Run 1.2i will help with making decisions what to 
rerun etc. Thanks again!

**Comment by cwwalter** [comment]
Add the following items which must be fixed before starting 2.0i:

- Write centroid files in .gz format (#149)
- Fix bleed trail wrap around (#153)
- Do GalSim 2.0 consistency check (#154).

**Comment by cwwalter** [comment]
Make a new sublist of issues (and a column for them in the Issue tracker board for issues primarily related to ANL workflow.  While not strictly necessary they are targeted to be done before we start 2.0i.

-  Estimate expected run times based on instance catalogs (#148)
-  Add an option to skip e-image production and only produce amp files (#146)
-  Write centroid files in .gz format (#149)
-  Make checkpoint summary information more easily available  to the workflow system (#147)

**Comment by cwwalter** [comment]
Due to the compressed time scale to start run 2.0i several of the outstanding imSim issues were merged at nearly the same time without all of the normal procedure.  They were all merged to a release candidate branch so we can unwind them if we find any issues and it is necessary. 

I will try to make a new issue where we can list all the work that needs to be merged into master (possibly just from that branch if things go well). This should allow us to more carefully understand what happened in the last week.

**Comment by cwwalter** [comment]
Forgot to add: some associated issues may remain open until we have safely landed them back on imSim master.

**Comment by cwwalter** [comment]
Ladies and Gentlemen,

Our pre-flight checklist is now complete and all of the necessary issue checkboxes are now filled.

At this time, make sure your seat backs and tray tables are in their full upright position and that your seat belt is correctly fastened. Also, your super-computing electronic devices must be set to ‘production’ mode until an announcement is made upon completion. Thank you, and from all of us on the imSim team we wish you a pleasant DC2 production experience.  

Everyone please prepare for takeoff.
